packages/flutter/lib/src/widgets/semantics_debugger.dart

```diff
@@ -22,13 +22,26 @@ class SemanticsDebugger extends StatefulWidget {
   /// Creates a widget that visualizes the semantics for the child.
   ///
   /// The [child] argument must not be null.
```

Comment:
Update this doc comment to include labelStyle?

Reply:
Done.